examples/image_list/lib/main.dart

```diff
@@ -133,6 +133,7 @@ class MyApp extends StatelessWidget {
       title: 'Flutter Demo',
       theme: ThemeData(
         primarySwatch: Colors.blue,
```

Comment:
Also here:)

Reply:
You mean that I should remove primarySwatch attribute ? if no, please provide a suggestion

Reply:
Yes, I think we can remove this line `primarySwatch: Colors.blue` because it looks redundant here. Material3 theme will provide a purple color as its primary color.